ssds/blobstore/local.py

```diff
@@ -73,8 +73,9 @@ def copy_from(self, src_blob: "LocalBlob"):
         if self.url != src_blob.url:
             shutil.copyfile(src_blob._path, self._path)
 
-    @catch_blob_not_found
     def download(self, path: str):
+        if not os.path.isfile(self._path):
+            raise BlobNotFoundError(f"Could not find {self.key}")
```

Comment:
Consider: f"Could not find `local` {self.key}"

Reply:
I'll change to `f"Could not find {self.url}"`, which prints the full path.